base/setup/lib/utils/partlist.c

```diff
@@ -673,6 +654,17 @@ CreateInsertBlankRegion(
     return NewPartEntry;
 }
 
+static
+VOID
+DestroyRegion(
+    _Inout_ PPARTENTRY PartEntry)
+{
+    // RemoveEntryList(&PartEntry->Volume->ListEntry);
```

Comment:
we cannot remove it (yet) because?

Reply:
The code is being designed and will continue to be after this PR, and things may change (I'm still considering whether I want to do it here instead of elsewhere).